project_parent_task_filter/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==========================
+Project parent task filter
+==========================
+
+This module adds a filter to show only the parent tasks 
+in a project.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Select the filter Parent tasks in a Project
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/11.0

```

Comment:
11.0 --> 12.0

Reply:
@tarteo I know `OCA-git-bot` is generated autometic via fragmented readme. but implicit better than explicit.